docs/core/build/distribution-packaging.md

```diff
@@ -17,49 +17,109 @@ ms.workload:
 
-## Disk layout
+- (9,10) **dotnet.1.gz, dotnet** `dotnet.1.gz` is the dotnet man page. `dotnet` is a symlink to the dotnet host(1). These files are installed at well known locations for system integration.
+
+## Recommended packages
+
+.NET Core versioning is based on the runtime component `[major].[minor]` version numbers.
+The SDK version uses the same `[major].[minor]` and has an independent `[patch]` which combines feature and patch semantics for the SDK.
+For example: SDK version 2.2.302 is a the 2nd patch release of the 3rd feature release of the SDK that supports the 2.2 runtime.
+
+Some of the packages include part of the version number in their name. This allows the end-user to install a specific version.
+The remainder of the version is not included in the version name. This allows the OS package manager to update the packages (e.g. automatically installing security fixes).
+
+The following tables shows the recommended packages.
+
+| Name                                    | Example                | Use case: Install ...           | Contains           | Dependencies                                   | Version            |
+|-----------------------------------------|------------------------|---------------------------------|--------------------|------------------------------------------------|--------------------|
+| dotnet-sdk-[major]                      | dotnet-sdk-2           | Latest sdk for runtime major    |                    | dotnet-sdk-[major].[latestminor]               | \<sdk version>     |
+| dotnet-sdk-[major].[minor]              | dotnet-sdk-2.1         | Latest sdk for specific runtime |                    | dotnet-sdk-[major].[minor].[latest sdk feat]xx | \<sdk version>     |
+| dotnet-sdk-[major].[minor].[sdk feat]xx | dotnet-sdk-2.1.3xx     | Specific sdk feature release    | (3),(4)            | aspnetcore-runtime-[major].[minor]             | \<sdk version>     |
+| aspnetcore-runtime-[major].[minor]      | aspnetcore-runtime-2.1 | Specific ASP.NET Core runtime   | (6),[(7)]          | dotnet-runtime-[major].[minor]                 | \<runtime version> |
+| dotnet-runtime-[major].[minor]          | dotnet-runtime-2.1     | Specific runtime                | (5)                | host-fxr:\<runtime version>+                   | \<runtime version> |
+| dotnet-host-fxr                         | dotnet-host-fxr        | _dependency_                    | (2)                | host:\<runtime version>+                       | \<runtime version> |
+| dotnet-host                             | dotnet-host            | _dependency_                    | (1),(8),(9),(10)   |                                                | \<runtime version> |
+
+Most distributions require all artifacts to be built from source. This has some impact on the packages:
+
+- The 3rd party libraries under `shared/Microsoft.AspNetCore.All` cannot be easily built from source. So that folder is omitted from the `aspnetcore-runtime` package.
```

Comment:
I don't have this folder (`shared/Microsoft.AspNetCore.All`) on my box. I also think this entire sentence is somewhat confusion and could use clarification.

Reply:
@JunTaoLuo why is there no `shared/Microsoft.AspNetCore.All` folder? Is the text wrong?

Reply:
@tannergooding daily cli builds have the `Microsoft.AspNetCore.All` folder. So the text is ok I think.